### ISSUE TITLE
[Feature Request] Single Shots

### ISSUE THREAD

**Issue by Ferrys93de** [Razeeman/Android-SimpleTimeTracker#235]
For things that could rather be "counted" than "tracked", but are still important in your day.
So, something like a counter with timestamps. 

That would allow a simple and effective tracking of
1. Taken in medication or supplements etc.
2. Consumption of coffee, water, sweets, etc.
3. Tracking of (good or bad) habits e.g. smoking etc.

**Comment by artiekra** [comment]
Is this planned to be added? I feel like this is the most important feature to add next, i myself have many uses for something like that

**Comment by Mehlsine** [comment]
There's a number of great open source apps that do exactly that, and do it well. Adding such a functionality in this project would sort of defy the philosophy of a "**simple** time tracker", don't you think?

https://github.com/SamAmco/track-and-graph
https://github.com/open-nomie/nomie6-oss

**Comment by dazu89** [comment]
I second this and disagree with @Mehlsine regarding alternatives (time tracking of track-and-graph is not convenient as end-time is not tracked) and contravening the philosophy of simplicity, as this could be added as a _simple_ custom field (or more general, one out of many) whose data could be saved in an additional spreadsheet column(s).

**Comment by Cancoillotte** [comment]
I use [uhabits](https://github.com/iSoron/uhabits) for this, which suits me fine. But it's true that there's no timestamp.

Otherwise, why not add periods without duration (it's possible, I sometimes accidentally add them)? And if necessary, allow several parallel periods?

**Comment by herrbrixter** [comment]
I agree with @Mehlsine . This suggestion doesn't feel in the spirit of this app.

Mixing points in time (without duration) with activities (which have durations) are very different and might introduce a lot of complexity in the code.

I use "[Chrono.me](https://play.google.com/store/apps/details?id=com.zagalaga.keeptrack)" Android app to track points in time. It's not open source but it's cheap and really well made.
